### PR TITLE
add languages

### PR DESCRIPTION
**Pull request by arowM**
#1

add following language supports
- C (clang, gcc)
- C++ (clang, g++)
- Go
- Groovy
- Haskell (GHC)
- OCaml
- Perl

@shunjikonishi please review.

### CODE REVIEW

**Comment by shunjikonishi** [comment]
@arowM fix me
In past, I used shell task to add something to .profile.  
But it can do with lineinfile task.

See. 

http://docs.ansible.com/lineinfile_module.html
https://github.com/givery-technology/code-env/blob/master/roles/envvars/tasks/main.yml

It is better to rewrite.

**Comment by arowM** [comment]
I changed to use lineinfile_module.
@shunjikonishi please review again.

**Comment by shunjikonishi** [comment]
I made a bit change to activator install task.
Because if I run ansible-playbook twice, it stop at unpackaging activator task.

@arowM review me

**Comment by arowM** [comment]
@shunjikonishi 
Could you tel me why you changed the privilege of download task from no-sudo to sudo?
There are regarding codes but they download files to `/tmp` without sudo.

**Comment by shunjikonishi** [comment]
I don't have deep reason.
If I'm forced to say, I don't like put user owned file in /usr/bin or /usr/share or something not user home directory.

File: roles/scala/tasks/activator.yml [hunk 1]
```diff
@@ -11,14 +11,14 @@
...
     dest=/tmp/{{activator_archive_file}}
     force=no
-  sudo: no
+  sudo: yes
 
 - name: Unpack activator
-  command: unzip {{activator_archive_file}} chdir=/tmp/
-  sudo: no
+  command: unzip {{activator_archive_file}} chdir=/tmp/ creates={{activator_archive_name}}
+  sudo: yes
```
Comment (arowM) [inline roles/scala/tasks/activator.yml]: Sorry for my lack of English ability, but I'd like to ask why you changed the parameter of `sudo` option from `no` to `yes`?
Is any deep reason?

**Comment by shunjikonishi** [comment]
@arowM fix me
I explained by word of mouth.
- Look again the task using /tmp
- Update README

**Comment by arowM** [comment]
- fix for issue #10 
- change regarding to previous comment
  @shunjikonishi please review

**Comment by shunjikonishi** [comment]
[LGTM](http://lgtm.in/i/SzohOIdPp)

**Pull request merged**